pkg/arduino_adafruit_sensor/Makefile.include

```diff
@@ -0,0 +1,2 @@
+CFLAGS += "-DARDUINO = 100"
```

Comment:
What does this do?

Reply:
It just says the source code of the package that we have an Arduino version > 1.0. This is usually set by the Arduino IDE.

Reply:
Mind adding a comment? Squashing out that `f` and triggering ci?